.flake8

```diff
@@ -1,3 +1,12 @@
 [flake8]
-max-line-length = 120
-ignore = E501
+# The following are ignored:
+#   E203 whitespace before ':' (https://github.com/ambv/black#line-breaks--binary-operators)
+#   E501 line too long         (https://github.com/ambv/black#line-length)
```

Comment:
What's the point of both setting the line length below *and* ignoring the warnings?

Reply:
Hmmmm. Black doesn't always change the line length. Some examples I've seen is long doc-strings and string parameters:
```
✗ black testing; flake8 testing | wc -l
All done! ✨ 🍰 ✨
22 files left unchanged.
     157
```